walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,115 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
+@media screen and (min-width: 600px)
+{
+  .header-container
+  {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    height: 100px;
+    width:100%;
+    max-width: 1000px;
+    margin-right: auto;
+    margin-left: auto;
+  }
 
-.image-thumbnails {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
-}
+  .header-item
+  {
+    text-decoration: none;
+    color: black;
+    font-size: 23px;
+  }
 
-.image-thumbnails img {
-  margin: 10px;
-  max-height: 200px;
-  max-width: 200px;
-}
+  .content-container
+  {
+    display: flex;
+    flex-direction: column;
+    border-style:dotted;
+    width:100%;
+    max-width: 1000px;
+    margin-right: auto;
+    margin-left: auto;
+  }
 
-.image-thumbnails img:hover {
-  border: thick solid black;
-}
+  .about-container
+  {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    border-style:solid;
+    width: 100%;
+  }
+
+  .description p
+  {
+    font-size: 19px;
+  }
+
+  img
+  {
+    width: 100%;
+    height: auto;
+    min-height: 300px;
+    min-width: 300px;
+  }
+
+  .footer-container
+  {
+    display: flex;
+    height: 200px;
+    width: 100%;
+    max-width: 1000px;
+    margin-right: auto;
+    margin-left: auto;
+  }
+
+  .column1
+  {
+    display: flex;
+    flex-direction: column;
+    width: 20%;
+    align-items: left;
+    justify-content: space-evenly;
+  }
+
+  .column1 a
+  {
+    text-decoration: none;
+    color: black;
+    font-size: 20px;
+    margin-left: 100px;
+  }
+
+  .column1 a:hover
+  {
+     text-decoration: underline;
+  }
+
+  .column2
+  {
+    display: flex;
+    flex-direction: column;
+    width: 80%;
+    align-items: center;
+    justify-content: space-evenly;
+    margin-left: 100px;
+  }
+
+  input
+  {
+    height:30px;
+    width:50%;
+    border-top:none;
+    border-bottom: solid 1px;
+    border-right: none;
+    border-left: none;
+    outline: none;
+  }
 
-#random-image-container img {
-  margin-top: 20px;
+  button
+  {
+    margin-top: 10px;
+    margin-bottom: 10px;
+    height:60px;;
+    width:50%;
+  }
 }
```

Comment:
It's generally preferred to put the base styles for html elements at the top of the file so that they can be overridden by classes after them.

Reply:
Also some of these attributes don't appear to be very generic. In the future if you add another button somewhere, would you want these styles applied to all of them? If not consider adding a class instead.

Reply:
Got it! Will do in future.

Reply:
Please do. I second this.
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css